src/tracker-component.js

```diff
@@ -381,12 +387,13 @@ export const Tracker = (config? : Config = defaultTrackerConfig) => {
         }
     };
     const emptyTrackers = {
-        addToCart:      (data : CartData) => doNoop(), // eslint-disable-line no-unused-vars
-        setCart:        (data : CartData) => doNoop(), // eslint-disable-line no-unused-vars
-        removeFromCart: (data : RemoveCartData) => doNoop(), // eslint-disable-line no-unused-vars
-        purchase:       (data : PurchaseData) => doNoop(), // eslint-disable-line no-unused-vars
-        setUser:        (data : UserData) => doNoop(), // eslint-disable-line no-unused-vars
-        setPropertyId:  (id : string) => doNoop(), // eslint-disable-line no-unused-vars
+        addToCart:      (data : CartData) => doNoop(),
+        setCart:        (data : CartData) => doNoop(),
+        removeFromCart: (data : RemoveCartData) => doNoop(),
+        purchase:       (data : PurchaseData) => doNoop(),
+        setUser:        (data : UserData) => doNoop(),
+        cancelCart:     (data : CancelCartData) => doNoop(),
+        setPropertyId:  (id : string) => doNoop(),
```

Comment:
can you add the // eslint-disable-line comments back? otherwise they result in errors when you do `npm run test`

Reply:
fixed